Week5/SOLUTION/week_5.py

```diff
@@ -0,0 +1,32 @@
+from csv import *
+
+header = ['SID','SNAME','CITY','CONTACT']
+
+file_record=[
+    [1,'Meet','Surat',1234567890],
+    [2,'Om','Bardoli',1213141516],
+    [3,'Sai','Vyara',8989890000],
+    [4,'Ram','Surat',1234500001],
+    [5,'Harsh','Surat',7227848152]
+    ]
+
+main_list=[]
+for i in range(5):
+    sid=int(input("Enter Student Id :"))
+    sname=input("Enter Student Name :")
+    scity=input("Enter Student City :")
+    scontact=int(input("Enter Student Contact :"))
+    sub_lis=[sid,sname,scity,scontact]
+    main_list.append(sub_lis)
+
+file_record.extend(main_list)

```

Comment:
CAN YOU PLEASE EXPLAIN ME WHAT IS ".extend(main_list)" HERE ?

Reply:
yes sure let take a example
list1 = [1,2,3]
list2 = [4,5]
list1.extend(list2) // it similar to append method but difference is append take one argument at a time and extend append one list to another list
print(list1)
---------output--------
[1,2,3,4,5]